### ISSUE TITLE
View a Proposal

### ISSUE THREAD

**Issue by parabolika66** [MyrionPhoenixmoon/scenarioo#41]
As a user I want to be able to look at a proposal so that I don't need to load the whole editor every time.

**Comment by adiherzog** [comment]
I agree, this is quite important. This should be the same proposal page / URL that can also be shared.
